apis/python/src/tiledbsoma/io/outgest.py

```diff
@@ -487,12 +487,7 @@ def _extract_obsm_or_varm(
     if num_cols is None:
         if soma_nd_array.tiledbsoma_has_upgraded_shape:
             num_cols = soma_nd_array.shape[1]
-        else:
-            try:
-                used_shape = soma_nd_array.used_shape()
-                num_cols = used_shape[1][1] + 1
-            except SOMAError:
-                pass  # We tried; moving on to next option
+        # Else: we tried; moving on to next option
```

Comment:
Optional: Combine the if statements with an "and", and either remove the comment or move it above the if statement and say what exactly "we tried".

Reply:
Thanks! :)